workspace/python/ds-pack/2.1.0-base-python-3.10.13/vscode-tunnel/scripts/init-or-start.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * The script aims to setup a vscode tunnel for the workspace. It does by starting a tunnel in the background.
+ * Usage - node /vscode-tunnel/scripts setup.js
+ */
+const { spawn } = require('child_process');
+const fs = require('fs');
+
+const outputLogFilePath = '/vscode-tunnel/logs/output.log';
+const errorLogFilePath = '/vscode-tunnel/logs/error.log';
+const scriptPath = '/vscode-tunnel/scripts/tunnel.sh';
+
+function sleep(milliseconds) {
+  const start = new Date().getTime();
+  while (new Date().getTime() - start < milliseconds);
```

Comment:
Should we use a non-CPU intensive sleep, as we have used at other places? Or do we want to intentionally keep it sync?

Reply:
Hey have intentionally kept it `sync` 😅 .